tests/test_crs.py

```diff
@@ -18,15 +16,15 @@
 # When possible, Rasterio gives you the CRS in the form of an EPSG code.
 def test_read_epsg(tmpdir):
     with rasterio.open('tests/data/RGB.byte.tif') as src:
-        assert src.crs == {'init': 'epsg:32618'}
+        assert src.crs.data == {'init': 'epsg:32618'}
```

Comment:
I'm finding that `crs.data` is predicated on the user knowing that `CRS` overrides `UserDict`.  What about adding a new property to CRS that returns `self.data`, so that in usages like this we can:

`assert src.crs.to_dict == {'init': 'epsg:32618'}`